pages/feed/index.tsx

```diff
@@ -290,7 +313,10 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
               selectedFilter={selectedRepo}
               setSelected={(repo) => {
                 if (!openSingleHighlight) {
-                  router.push(`/feed${repo ? `?repo=${repo}` : ""}`);
+                  const queryParams = new URLSearchParams(window.location.search);
```

Comment:
Here too

Reply:
Didn't get this, can you add a code snippet please?

Reply:
Here @diivi 

```js
router.push(
  `/feed${repo ? router.asPath `&repo=${repo}` : router.asPath ? router.asPath : ""}`
);
```